spaceship/docs/AppStoreConnect.md

```diff
@@ -358,7 +358,7 @@ impressions = analytics.app_impressions  # => Array of dates representing raw da
 sales = analytics.app_sales              # => Array of dates representing raw data for each day
 
 # Get paying users
-users = analytics.paying_users           # => Array of dates representing raw data for each day
+users = analytics.app_paying_users       # => Array of dates representing raw data for each day
```

Comment:
was this wrong before?

Reply:
According to below code it might mistyped [when](https://github.com/fastlane/fastlane/pull/14167) documents added.
https://github.com/fastlane/fastlane/blob/2.149.0/spaceship/lib/spaceship/tunes/app_analytics.rb#L51